### PR TITLE
Use progress element

### PR DESCRIPTION
**Pull request by camillobruni**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -41,7 +41,7 @@
                 </a>
                 <div id="testContainer"></div>
                 <div id="progress">
-                    <div id="progress-completed"></div>
+                    <progress id="progress-completed"></progress>
```
Comment (bgrins) [inline index.html:44]: I think this should have an `aria-label` or be wrapped inside a `<label>` element as per https://developer.mozilla.org/en-US/docs/Web/HTML/Element/progress#labelling

File: resources/main.css [hunk 2]
```diff
@@ -310,7 +309,17 @@ button.show-about {
...
+    appearance: none;
+    border: none;
+    background-color: var(--inactive-color);
+}
+
+#progress-completed::-webkit-progress-value {
+    background-color: var(--foreground);
+}
+
+#progress-completed::-moz-progress-bar {
```
Comment (flashdesignory) [inline resources/main.css:322]: any way to make the progress smoother? 
maybe something simple like:
`transition: all 0.3s linear;` 🤷

Comment (julienw) [reply resources/main.css:322]: why not but it's not 100% needed as usually the steps are short in width, so it's not hugely visible anyway...

File: index.html (hunk 1 above)
Comment (camillobruni) [reply index.html:44]: Added `aria-label`, thanks for the review.

File: resources/main.css (hunk 2 above)
Comment (camillobruni) [reply resources/main.css:329]: I think it want to have as little work as possible happening between the benchmark steps.
I'd prefer not having animations (except in workloads maybe).

**Pull request merged**